src/System.IO/ref/4.1/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "System.Runtime": "4.0.20",
```

Comment:
This needs to be only "System.IO" "4.1.0" so bin-placing works as expected. This is why the CI legs are failing.